### ISSUE TITLE
Non-localized string in CheckFeatureAvailability (VB)

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#17807]
```VB
        Friend Shared Function CheckFeatureAvailability(Of TNode As VisualBasicSyntaxNode)(feature As Feature, node As TNode, languageVersion As LanguageVersion) As TNode
            If CheckFeatureAvailability(languageVersion, feature) Then
                Return node
            End If

            If feature = Feature.InterpolatedStrings Then
                ' Bug: It is too late in the release cycle to update localized strings.  As a short term measure we will output 
                ' an unlocalized string and fix this to be localized in the next release.
                Dim requiredVersion = New VisualBasicRequiredLanguageVersion(feature.GetLanguageVersion())
                Return ReportSyntaxError(node, ERRID.ERR_LanguageVersion, languageVersion.GetErrorName(), "interpolated strings", requiredVersion)
            Else
                Return ReportFeatureUnavailable(feature, node, languageVersion)
            End If
        End Function

```

**Comment by AdamSpeight2008** [comment]
@jcouv Re: #17761 fix implemented in #17751

**Comment by jcouv** [comment]
Thanks.
Closing this bug as duplicate of https://github.com/dotnet/roslyn/issues/17761
